doltpy/etl/loaders.py

```diff
@@ -294,11 +337,9 @@ def create_table_from_schema_import(repo: Dolt,
     _create_table_from_schema_import_helper(repo, table, pks, path, commit=commit, commit_message=commit_message)
 
 
-def create_table_from_schema_import_unique_key(repo: Dolt,
-                                               table: str,
-                                               path: str,
-                                               commit: bool = True,
-                                               commit_message: str = None):
+def create_table_from_schema_import_unique_key(
+    repo: Dolt, table: str, path: str, commit: bool = True, commit_message: str = None
```

Comment:
Seems this should be multi line?

Reply:
yep, i agree. black defaults to the simplest change to get a line syntax-compliant, can't always trust its output to be the nicest-looking